src/web.ts

```diff
@@ -1,22 +1,61 @@
-import { WebPlugin } from '@capacitor/core';
-import { SignInWithApplePlugin } from './definitions';
+import { WebPlugin } from "@capacitor/core";
 
-export class SignInWithAppleWeb extends WebPlugin implements SignInWithApplePlugin {
+import { SignInWithApplePlugin, InitOptions } from "./definitions";
+
+declare var window: any;
+
+export class SignInWithAppleWeb extends WebPlugin
```

Comment:
Why is the public API first-letter capitalised? I know the iOS plugin has followed this convention for some reason, but would it be worth camel-casing as per...everything else... (including the iOS API) and then having a major-version bump to reflect the breaking change?

Reply:
I've made a separate issue for this https://github.com/capacitor-community/apple-sign-in/issues/5 as @priyankpat rightly pointed out that it's outside the scope of this PR